src/components/FeaturePanel/EditDialog/EditContent/AddMemberForm.tsx

```diff
@@ -39,22 +39,28 @@ const getNewNodeLocation = async (items: EditDataItem[], members: Members) => {
   return lonLat.map((x) => x + 0.0001);
 };
 
-export const AddMemberForm = () => {
-  const { addFeature, items } = useEditContext();
-  const { members, setMembers } = useFeatureEditData();
+export const AddMemberForm = ({ newLonLat }: { newLonLat?: LonLat }) => {
+  const { addFeature, items, setCurrent, current } = useEditContext();
+  const { members, setMembers, tags } = useFeatureEditData();
   const [showInput, setShowInput] = React.useState(false);
   const [label, setLabel] = React.useState('');
+  const isClimbingCrag = tags.climbing === 'crag';
 
-  if (!hasAtLeastOneNode(members)) {
+  if (!hasAtLeastOneNode(members) && !newLonLat) {
     return; // TODO so far, we need a node (with coordinates) for adding a new node
   }
 
   const handleAddMember = async () => {
-    const lastNodeLocation = await getNewNodeLocation(items, members);
+    console.log('___!!', items);
+    if (isClimbingCrag) {
+      setCurrent(current.replace(/n/, 'r'));
```

Comment:
This is not enough, we have to change also items. @zbycz any ideas how to do it?